### PR TITLE
refactor: unify dml and dql handler path

### PR DESCRIPTION
**Pull request by BowenXiao1999**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

Refactor to make dml and dql use the same path.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

* Installation and deployment 
* Connector (sources & sinks)
* SQL commands, functions, and operators
* RisingWave cluster configuration changes
* Other (please specify in the release note below)

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

File: src/frontend/src/handler/query.rs [hunk 1]
```diff
@@ -80,9 +88,23 @@ pub async fn handle_query(
...
+                .unwrap()
+                .parse()
+                .unwrap_or_default()
+        }
         _ => unreachable!(),
     };
 
+    // Implicitly flush the writes.
+    if session.config().get_implicit_flush() {
+        dml::flush_for_write(&session, stmt_type).await?;
```
Comment (BugenZhao) [inline src/frontend/src/handler/query.rs:106]: Can we also move this function out of `dml.rs`?

**Review by BugenZhao — APPROVED** [review]
Glad to see this clean-up! 😄

File: src/frontend/src/handler/query.rs (hunk 1 above)
Comment (BowenXiao1999) [reply src/frontend/src/handler/query.rs:106]: I was thinking. But indeed flush_for_write is something relates to dml..

Comment (BowenXiao1999) [reply src/frontend/src/handler/query.rs]: Fixed

**Pull request merged**